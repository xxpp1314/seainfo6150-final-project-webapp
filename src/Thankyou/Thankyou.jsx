import React from 'react';
import styles from "./Thankyou.module.css";

const Thankyou = () => {
    return (
        <div className={styles.grid}>
            <div className={styles.thankyou}>
                <h3>We have received your information.</h3><br />
                <h1>Thank you for your time. We will contact you soon! </h1>
            </div>
        </div>
    )
}

export default Thankyou;