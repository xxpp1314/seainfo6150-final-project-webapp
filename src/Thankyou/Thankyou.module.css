.thankyou{
    padding: 15%;
    text-align: center;
    color:  rgb(201, 187, 187);
    box-shadow: 0px 0px 10px 0px rgba(204, 0, 0, 0.5);    
    border: 0px solid #666;
    display: flex;
    flex-direction: column;
    align-items: center;
}

.grid{
    display: grid;
    grid-template-columns: 1fr;
    align-items: center;
    background-image: linear-gradient(to right,rgb(83, 15, 15), rgb(49, 12, 25) 70%);
    padding: 5%;
    box-shadow: 0px 0px 10px 0px rgba(204, 0, 0, 0.5);
}