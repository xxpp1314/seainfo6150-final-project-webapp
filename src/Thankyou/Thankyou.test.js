import React from "react";
import { render } from "@testing-library/react";
import Thankyou from "./Thankyou.jsx";

it("renders the Thank You Page correctly", () => {
  const { container } = render(<Thankyou />);
  expect(container).toMatchSnapshot();
});
