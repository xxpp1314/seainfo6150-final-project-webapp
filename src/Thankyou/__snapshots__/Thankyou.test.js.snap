// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Thank You Page correctly 1`] = `
<div>
  <div
    class="grid"
  >
    <div
      class="thankyou"
    >
      <h3>
        We have received your information.
      </h3>
      <br />
      <h1>
        Thank you for your time. We will contact you soon! 
      </h1>
    </div>
  </div>
</div>
`;
